__tests__/pages/resource/id/evaluate.test.tsx

```diff
@@ -322,7 +327,9 @@ describe("Select component, Radio button, and request preview render", () => {
     //expect the request preview to include reportType=population
     expect(
       screen.getByText(
-        "/Measure/Measure-12/$evaluate-measure?periodStart=2022-01-01&periodEnd=2022-12-31&reportType=population",
+        `/Measure/Measure-12/$evaluate-measure?periodStart=${DateTime.now().year}-01-01&periodEnd=${
```

Comment:
We now have you to thank for when the unit tests don't mysteriously break in 2023 lol

Reply:
I realized I forgot to make this fix in MeasureDatePickers.test.tsx so I just added it